docs/copilot/copilot-customization.md

```diff
@@ -139,6 +139,64 @@ To configure custom commit message generation instructions, use the `setting(git
 * `setting(github.copilot.chat.reviewSelection.instructions)` _(Preview)_: A set of instructions that will be added to Copilot requests for reviewing the current editor selection.
 * `setting(github.copilot.chat.commitMessageGeneration.instructions)` <i class="codicon codicon-beaker"></i>: A set of instructions that will be added to Copilot requests that generate commit messages.
 
+## Reusable Prompt Files (Preview)
+
+Prompt Files in GitHub Copilot Chat are designed to streamline workflows, enhance collaboration, and bring domain-specific expertise to you and your team's fingertips. With reusable and shareable prompts, you can save time, maintain consistency, and tackle complex tasks with ease. While the existing file-based custom instructions help to add codebase-wide context to each AI workflow, developers can add specific prompts as they need them, which makes them scale to the complexity of larger multi-faceted codebases.
```

Comment:
- Suggest to drop some of the marketing language - might still be useful for the blog.
- Lead with what Prompt files are and focus on the capabilities.